### ISSUE TITLE
Agent can not go up stairs

### ISSUE THREAD

**Issue by tesslerc** [Microsoft/malmo#491]
Running with discrete movements, agent doesn't auto walk up stairs.

**Comment by DaveyBiggers** [comment]
Have you set `autoJump`? It's false by default.
(You might also want `autoFall`.)

**Comment by tesslerc** [comment]
Didn't set autoJump because I assume it will also cause the agent to jump over other blocks/fences.

**Comment by DaveyBiggers** [comment]
Good point. Perhaps stairs should behave as if autoJump/autoFall are on.
As a workaround, you can make sure everything else is at least two blocks high - it will only jump up one block at a time.

**Comment by tesslerc** [comment]
I did a different workaround.
It works but isn't optimal.
If block in-front is 'steps' and action is move -> action is changed to jumpmove
